pub_sub/java/sdk/checkout/src/main/java/com/service/CheckoutServiceApplication.java

```diff
@@ -0,0 +1,28 @@
+package com.service;
+
+import io.dapr.client.DaprClient;
+import io.dapr.client.DaprClientBuilder;
+import io.dapr.client.domain.Metadata;
+
+import static java.util.Collections.singletonMap;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+public class CheckoutServiceApplication {
+	public static void main(String[] args) throws InterruptedException{
```

Comment:
I like it that you have this in a single sub main vs. refactoring into several functions (which I've seen in other braches).  Please continue with this level of simplicity.